test/e2e/suite/issuers/acme/certificate/http01.go

```diff
@@ -271,6 +274,62 @@ var _ = framework.CertManagerDescribe("ACME Certificate (HTTP01)", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
+	It("should obtain two separate signed certificates with the same CN rom the ACME server when creating two annotated ingress resources", func() {
+
+		switch {
+		case util.HasIngresses(f.KubeClientSet.Discovery(), networkingv1.SchemeGroupVersion.String()):
+			ingClient := f.KubeClientSet.NetworkingV1().Ingresses(f.Namespace.Name)
+			By("Creating two Ingresses with the issuer name annotation set")
+			_, err := ingClient.Create(context.TODO(), util.NewIngress(ingressName, certificateSecretName, map[string]string{
+				"cert-manager.io/issuer": issuerName,
+			}, acmeIngressDomain), metav1.CreateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			ingress2 := util.NewIngress(ingressName2, certificateSecondSecretName, map[string]string{
+				"cert-manager.io/issuer": issuerName}, acmeIngressDomain)
+			ingress2.Spec.Rules[0].IngressRuleValue.HTTP.Paths[0].Path = "/different"
+
+			_, err = ingClient.Create(context.TODO(), ingress2, metav1.CreateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+		case util.HasIngresses(f.KubeClientSet.Discovery(), networkingv1beta1.SchemeGroupVersion.String()):
```

Comment:
I don't think we have to also support networkingv1beta1 in our tests. I suspect that all kubernetes versions that we use use networkingv1.

Reply:
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingressclass-v122

> The networking.k8s.io/v1beta1 API version of IngressClass is no longer served as of v1.22.

https://cert-manager.io/docs/releases/#currently-supported-releases


Release | Release Date | End of Life | [Supported Kubernetes versions](https://cert-manager.io/docs/releases/#kubernetes-supported-versions) | [Supported OpenShift versions](https://cert-manager.io/docs/releases/#kubernetes-supported-versions)
-- | -- | -- | -- | --
[1.15](https://cert-manager.io/docs/releases/release-notes/release-notes-1.15/) | Jun 05, 2024 | Release of 1.17 | 1.24 → 1.30 | 4.11 → 4.15
[1.14](https://cert-manager.io/docs/releases/release-notes/release-notes-1.14/) | Feb 03, 2024 | Release of 1.16 | 1.24 → 1.30 | 4.11 → 4.15